iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/AbstractCreateConnectableFeederBays.java

```diff
@@ -94,7 +94,7 @@ public void apply(Network network, boolean throwException, ComputationManager co
             VoltageLevel voltageLevel = getVoltageLevel(side, connectable);
             Set<Integer> takenFeederPositions = TopologyModificationUtils.getFeederPositions(voltageLevel);
             int positionOrder = getPositionOrder(side);
-            if (!takenFeederPositions.isEmpty() || voltageLevel.getConnectableStream().count() == 1) {
+            if (!takenFeederPositions.isEmpty() || (voltageLevel.getConnectableStream().count() == 1 + voltageLevel.getNodeBreakerView().getBusbarSectionCount())) {
```

Comment:
Use this check instead: `voltageLevel.getConnectableStream().filter(c -> !(c instanceOf BusbarSection)).count() == 1`